.github/CONTRIBUTING.md

```diff
@@ -1,388 +1,97 @@
 
 ## Start contributing
 
 ### 1. Fork the repo:
 
-Go to [carbon-components](https://github.com/IBM/carbon-components) and click
-the `Fork` button in the top-right corner.
+Go to
+[Carbon's repository on GitHub](https://github.com/carbon-design-system/carbon)
+and click the `Fork` button in the top-right corner. This will create a copy
+repo of Carbon associated with your account.
 
 ### 2. Clone your fork:
 
 1.  Go to your [GitHub Repositories](https://github.com/settings/repositories).
-1.  Click on `[your_github_username]/carbon-components`.
+1.  Click on `[your_github_username]/carbon`.
 1.  Click on the `Clone or Download` button and copy the URL from the
     `Clone with SSH` option. It should start with `git@github.com...`
 
 In your terminal:
 
 ```sh
-git clone git@github.com:[your_github_username]/carbon-components.git
-cd carbon-components
+git clone git@github.com:[your_github_username]/carbon.git
+cd carbon
 ```
 
 See [GitHub docs](https://help.github.com/articles/fork-a-repo/) for more
 details.
 
 ### 3. Add upstream remotes
 
-When you clone your forked repo, doing a `git remote -v` will show that the
-`origin` remote is set up for you already by default. This should be pointing to
-your forked repo.
+When you clone your forked repo, running `git remote -v` will show that the
+`origin` is pointing to your forked repo by default.
 
-Add the `IBM/carbon-components` repo to your remote (this can be useful to
-update your fork of new changes down the road):
+Now you need to add the `carbon-design-system/carbon` repo as your upstream
+remote branch:
 
 ```sh
 # Add the upstream remote to your repo
-git remote add upstream git@github.com:IBM/carbon-components.git
+git remote add upstream git@github.com:carbon-design-system/carbon.git
 
 # Verify the remote was added
 git remote -v
 ```
 
-When you do `git remote -v`, you'll see these remotes:
+Your terminal should output something like this:
 
-- `origin`: connection to your fork
-- `upstream`: connection to the original repo.
+```sh
+origin  [your forked repo] (fetch)
+origin  [your forked repo] (push)
+upstream    git@github.com:carbon-design-system/carbon.git (fetch)
+upstream    git@github.com:carbon-design-system/carbon.git (push)
+```
 
 ### 4. Work in a branch
 
-- Always work in a branch.
-- Submit pull requests from a branch.
-- All commits must follow the convention outlined
-  [here](https://github.com/conventional-changelog/conventional-changelog/blob/v0.5.3/conventions/angular.md).
+When contributing to Carbon, your work should always be done in a branch off of
+your repo, this is also how you will submit your pull request when your work is
+done.
 
-### 5. Start the server
+To create a new branch, ensure you are in your forked branch in your terminal
+and run:
 
 ```sh
-npm run dev
+$ git pull origin master
+$ git checkout -b {your-branch-name}
+```
+
+### 5. Start the development server
 
-# or
+To start your development server, run:
+
+```sh
 
 yarn dev
```

Comment:
Running `yarn dev` from the root does nothing. Additionally, it would be helpful to outline that in addition to running `yarn`, you must also run `yarn build` before things will work. This is in the developer handbook, but would be helpful here as well.

Reply:
Good catch! Thanks for looking into this! I added more details around the build and starting the development server if you would like to check them out.

Reply:
Running `yarn build` should be called out.

I had trouble setting up my environment because I hadn't run `yarn build` at the root first. I was getting the "Can't resolve X..." when trying to run the react Storybook, even after a fresh clone and install.